lib/common/huf.h

```diff
@@ -82,20 +87,22 @@ HUF_PUBLIC_API const char* HUF_getErrorName(size_t code);  /**< provides error c
 /** HUF_compress2() :
  *  Same as HUF_compress(), but offers control over `maxSymbolValue` and `tableLog`.
  * `maxSymbolValue` must be <= HUF_SYMBOLVALUE_MAX .
- * `tableLog` must be `<= HUF_TABLELOG_MAX` . */
+ * `tableLog` must be `<= HUF_TABLELOG_MAX` . 
+ *  If depthStrategy = HUF_seek_neighbor additional huffman depths will be considered */
 HUF_PUBLIC_API size_t HUF_compress2 (void* dst, size_t dstCapacity,
                                const void* src, size_t srcSize,
-                               unsigned maxSymbolValue, unsigned tableLog);
+                               unsigned maxSymbolValue, unsigned tableLog, HUF_depthStrategy depthStrategy);
 
 /** HUF_compress4X_wksp() :
  *  Same as HUF_compress2(), but uses externally allocated `workSpace`.
- * `workspace` must be at least as large as HUF_WORKSPACE_SIZE */
+ * `workspace` must be at least as large as HUF_WORKSPACE_SIZE 
+ *  If depthStrategy = HUF_seek_neighbor additional huffman depths will be considered */
 #define HUF_WORKSPACE_SIZE ((8 << 10) + 512 /* sorting scratch space */)
 #define HUF_WORKSPACE_SIZE_U64 (HUF_WORKSPACE_SIZE / sizeof(U64))
 HUF_PUBLIC_API size_t HUF_compress4X_wksp (void* dst, size_t dstCapacity,
                                      const void* src, size_t srcSize,
                                      unsigned maxSymbolValue, unsigned tableLog,
-                                     void* workSpace, size_t wkspSize);
+                                     void* workSpace, size_t wkspSize, HUF_depthStrategy depthStrategy);
```

Comment:
Parameters order consistency : 
`workSpace` is usually the last parameter of `_wksp()` variants.
Keep it that way.
Having `depthStrategy` after `tableLog` preserves parameters order compared to `HUF_compress2()`.